docs/config.adoc

```diff
@@ -2307,6 +2308,59 @@ when activated by the override which is important to consider for your use cases
 )
 ----
 
+[[allow-hardware-repeat]]
+=== allow-hardware-repeat
+<<table-of-contents,Back to ToC>>
+
+By default, any repeat-key events generated by the physical keyboard (or operating system)
+will be passed through to the application.  On Linux, this is known to cause some problems;
+see https://github.com/jtroo/kanata/discussions/422[discussion #422] and https://github.com/jtroo/kanata/issues/450[issue #450].
+
+Setting this option to "false" will cause such events to be dropped, and not passed through.
+This is primarily meant for Linux, but may find some use on Mac.  It is not implemented on
+Windows, and will be silently ignored.
+
+.Example:
+[source]
+----
+(defcfg
+   allow-hardware-repeat false
+)
+----
+
+[[start-alias]]
```

Comment:
The naming doesn't feel explicit enough for what this does.

I would prefer something like `alias-to-trigger-on-load`. It is configured only 0 or 1 times; given the infrequency of being both read and written, being explicit seems better.

Reply:
Done, and pushed.